.gitignore

```diff
@@ -171,3 +171,4 @@ scratch*
 
 # Playwright test automation
 playwright-report/
+*TEST_*
```

Comment:
Does this also ignore the test files under `mage_ai/tests`?

Reply:
No, since that's case sensitive. I can update to *TEST*.json to be more careful

Reply:
I tested locally. There's a git config `git config core.ignorecase`. If it's true, then it ignore the case. My default setting is true somehow.

Reply:
huh, odd. I've already accidentally uploaded gh tokens twice by following our guide, so I'll be explicit with file names.